utils/constants.py

```diff
@@ -14,6 +14,8 @@
 ARRAY_MIN = 1
 ARRAY_MAX = 5
 
+ZERO_UUID = "00000000-0000-0000-0000-000000000000"
```

Comment:
why do we need zero/ nil uuid?

Reply:
good question! Because we now validate that the ID fields are empty in some endpoints, when we pass it in, it either needs to be a blank but valid UUID field (so the zero UUID), or it needs to be omitted from the payload completely. We unfortunately can't mark these fields as `readOnly` just yet because the same definition is used for the update endpoints, and those endpoints require the ID. It's also a huge pain to try and pop these fields from the dictionary when they're in nested objects as well. 

So the `ZERO_UUID` value ends up being the easiest solution (and this is the value that gets set for a nil UUID in `payload_to_model` so the end result is the same either way).